features/FooterUpdate.feature

```diff
@@ -0,0 +1,66 @@
+@owner:derek
+@author:lilla
+Feature: Docs Footer Update
+
+    New responsive design and links for the footer on the docs pages.
+
+    Background: A user arrives on the SaaSquatch help center page
+        Given the user stays on the 'https://docs.saasquatch.com/' pages
+        Then the footer categories are visible
+
+
+    @motivating
+    Scenario: Categories and links for the footer
+        Given the user looks at the footer
+        Then the following categories and links are visible
+        And the user can click on them
+            | Product   | Login             | https://app.referralsaasquatch.com                                            |
+            | Product   | Customer Loyalty  | https://www.saasquatch.com/loyalty-program-software/                          |
+            | Product   | Customer Referral | https://www.saasquatch.com/referral-program-software/                         |
+            | Product   | Request Demo      | https://www.saasquatch.com/request-demo/                                      |
+            | Product   | Integrations      | https://www.saasquatch.com/api-integrations/                                  |
+            | Product   | Status Page       | https://status.saasquatch.com/                                                |
+            | Company   | About             | https://www.saasquatch.com/about/                                             |
+            | Company   | Partner Program   | https://www.saasquatch.com/partners/                                          |
+            | Company   | Careers           | https://www.saasquatch.com/careers/                                           |
+            | Company   | Contact Us        | https://www.saasquatch.com/contact/                                           |
+            | Resources | Blog              | https://www.saasquatch.com/blog/                                              |
+            | Resources | Podcast           | https://www.saasquatch.com/the-advocacy-channel-a-customer-marketing-podcast/ |
+            | Resources | Case Studies      | https://www.saasquatch.com/customers/                                         |
+            | Resources | Loyalty Academy   | https://www.digitalloyaltyacademy.com/                                        |
+            | Legal     | Terms of Service  | https://www.saasquatch.com/terms-of-service/                                  |
+            | Legal     | Privacy Policy    | https://www.saasquatch.com/privacy-policy/                                    |
+            | Legal     | Terms of Use      | https://www.saasquatch.com/terms-of-use/                                      |
+
+    @motivating
+    Scenario: At the bottom of the footer there is copywrite and address information
+        Given the user looks at the bottom of the footer
+        Then the following information is visible
```

Comment:
Our standard way of saying that something is visible to a user is, `They see the following information` try to apply this type of wording to the other cases like this as well